contracts/contracts/VerificationGateway.sol

```diff
@@ -361,4 +363,7 @@ contract VerificationGateway
         );
     }
 
+     function domainSeparator() public view returns (bytes32) {
```

Comment:
If this is not going to be used inside the contract, such as at the `_domainSeparatorV4` callpoints above, it can be marked as external. In this case it doesn't matter much, but see https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices for a reason why it can make a difference.

```suggestion
     function domainSeparator() external view returns (bytes32) {
```